lib/reek/cli/utilities.rb

```diff
@@ -0,0 +1,12 @@
+module Reek
+  module CLI
+    # CLI utilities
+    module Utilities
+      def self.require_silently
+        old_verbose, $VERBOSE = $VERBOSE, nil
+        yield if block_given?
+        $VERBOSE = old_verbose
+      end
```

Comment:
I’m probably way over-zealous in this case, but can we put the reset line in an `ensure` block?

```ruby
      def self.require_silently
        old_verbose, $VERBOSE = $VERBOSE, nil
        yield if block_given?
      ensure
        $VERBOSE = old_verbose
      end
```

(We _could_ try making this thread-safe, but that would be overdoing it even for my tastes.) ;)